pkg/federation/sdlmerge/interface_type_extending.go

```diff
@@ -47,3 +58,25 @@ func (e *extendInterfaceTypeDefinitionVisitor) EnterInterfaceTypeExtension(ref i
 		e.Walker.StopWithExternalErr(operationreport.ErrExtensionOrphansMustResolveInSupergraph(e.document.InterfaceTypeExtensionNameBytes(ref)))
 	}
 }
+
+func (e extendInterfaceTypeDefinitionVisitor) getWalker() *astvisitor.Walker {
+	return e.Walker
+}
+
+func (e extendInterfaceTypeDefinitionVisitor) getDocument() *ast.Document {
```

Comment:
try to not mix pointer and value receiver. If unsure, stick with pointer receiver

Reply:
So all receivers should be either pointers or values, but never a mix?

Reply:
I think GoLand creates missing methods with pointer receivers, but I don't think they're necessary, because the document and Walker are both pointers.

Reply:
In case you will have a non pointer receiver each time you will visit this method you will get a copy of a struct